index.html

```diff
@@ -1,297 +1,46 @@
-              </p>
-        </section>
-
-        <section class="main-content" id="venue">
-              <h2 class="lead"> Venue </h2>
-              <h3> Host Venue </h3>
-              <p> Jaypee Institute of Information Technology <br>
-                  Other Cybersristi events organized at host venue:
-                  <a href="http://www.jiit.ac.in/jiit/cybersrishti/website.html" target="_">http://www.jiit.ac.in/jiit/cybersrishti/website.html</a></p>
-              <h3> Address </h3>
-              <p> Jaypee Institute of Information Technology <br>
-                A-10,Sector-62 <br>
-                Noida-201 307<br>
-                Uttar Pradesh 201307
-              </p>
-              <h5> LandMark </h5>
-              <span id="landmark"> Near Noida Expo Center </span>
-              <div id="map-canvas"/></div>
-        </section>
-      </div>
+    <div class="overlay">
+      <img src="assets/images/osdpng.png" height="229px" width="358.5px">
+      <form action="main.html">
+        <button class="proceed" type="submit">Jump In</button>
+      </form>
     </div>
-    </div>
-    </main>
-    <footer>
-        <hr>
-        <div id="osdc-logo">
-            <a href="http://osdc.github.io" target='_blank'><img src="img/logo.png" alt="logo"></a>
-        </div>
-    </footer>
-
-
-    <script type="text/javascript"
-      src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCtnTt4diLgTb8nC0P9FCFttB2GgdHLGqk&sensor=true">
-    </script>
-    <script type="text/javascript">
-      function initialize() {
-        var JIIT = new google.maps.LatLng(28.630298, 77.372050);
-        var mapOptions = {
-          center: JIIT,
-          zoom: 15,
-          mapTypeId: google.maps.MapTypeId.ROADMAP
-        };
-        var map = new google.maps.Map(document.getElementById("map-canvas"),
-            mapOptions);
 
-        var marker= new google.maps.Marker({
-          position: JIIT,
-          map: map,
-          title: "Jaypee Institute of Information Technology"
+    <div class="screen">
+      <div class="navbar">
+          <button class="close">
+          </button>
+          <button class="minimize">
+          </button>
+          <button class="maximize">
+          </button>
+          <span class="session">/home/osdconf/</span>
+      </div>
 
-        });
+      <div class="terminal">
+        <p class="line1">$ Welcome to OSDCONF<span class="cursor1">_</span></p>
+        <p class="line2">&nbsp;>Loading assets...<span class="cursor2">_</span></p>
+        <p class="line3">$ loop info<span class="cursor2">_</span></p>
+        <p class="line4">&nbsp;> Permission denied</p>
+        <p class="line5">$ sudo loop info<span class="cursor4">_</span></p>
+        <div id="line3"></div>        
+      </div>
+  </div>
+  <button class="skip"><a href="main.html"><i class="fa fa-fast-forward" aria-hidden="true"></i></a></button>
```

Comment:
Why doesn't this button work?

Reply:
I have not pushed main.html yet. It will work once I add those files.